src/kOS/LightColor/kOSLiteModule.cs

```diff
@@ -0,0 +1,90 @@
+using System;
+using UnityEngine;
+using KSP.IO;
+
+namespace kOS.LightColor
+{
+    class kOSLiteModule : PartModule
```

Comment:
Pure pedantry, I know, but spelling it "kOSLiteModule" makes it sound like a stripped-down smaller footprint version of kOS, rather than a light as in illumination.  I.e. "lite" as in "lite beer", not "light" as in "turn on the light".

At any rate, it should probably match the namespace spelling one way or the other.  Right now it's a class called "lite" stored in a namespace called "light".